src/App.tsx

```diff
@@ -1,7 +1,33 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  key: string;
+};
+
+export class App extends React.Component<{}, State> {
+  state = {
+    key: '',
+  };
+
+  componentDidMount() {
+    document.addEventListener('keyup', (e) => {
```

Comment:
couple tusks ago i got a comment about naming variable inside function with one later 'e', 'n' etc. not the best practice

Reply:
In a lot of key `e` - it's ok, but let's put `event` name here